### DISCUSSION TITLE
gesture: Hey are you giveing me the bird in JScode?

### DISCUSSION THREAD

**Discussion by meeki007** [vladmandic/human#76] [General]
So I'm going through ALL the documentation. Just reading along and I come upon the last section of Gesture on the WiKi.

```JS
{hand: "0", gesture: "thumb forward middlefinger up"}
```

* in a joking tone - Did you keep a copy of this image? if so I claim the Easter egg I found.

**Comment by vladmandic** [comment]
LOL :)  
No, don't have the image, testing was done with a webcam :)
